infra/recipes/terraform/azure/sandbox/storage-services.tf

```diff
@@ -0,0 +1,7 @@
+resource "azurerm_storage_account" "sandbox" {
```

Comment:
Where this resource is used?

Reply:
It isn't here. However, it is more a proof of concept because we will need it when we deploy the actual sandbox.